crates/libs/nuts/src/nut00/mod.rs

```diff
@@ -65,6 +66,10 @@ pub struct Proof {
     /// Unblind signature
     #[serde(rename = "C")]
     pub c: PublicKey,
+    /// Optional DLEQ proof for validating the unblinded signature offline (NUT-12).
+    #[cfg(feature = "nut12")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub dleq: Option<DleqProof>,
```

Comment:
It should not be optional.
It is optional in the protocol, meaning that node that don't support nut12 don't have to put it.
But if a node support nut12 (and therefore enable this feature) it HAS to fill this field, it cannot leave it empty